src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -172,6 +172,17 @@ def k8s_install_cli(client_version="1.4.5", install_location=None):
     except IOError as err:
         raise CLIError('Connection error while attempting to download client ({})'.format(err))
 
+def _validate_service_principal(sp_id):
+    from azure.cli.command_modules.role.custom import (
+        _graph_client_factory,
+        show_service_principal,
+    )
```

Comment:
Related to this from a prev. PR - https://github.com/Azure/azure-cli/pull/1258#discussion_r86880289